milli/src/search/mod.rs

```diff
@@ -241,7 +254,7 @@ pub fn word_derivations<'c>(
             }
 
             Ok(entry.insert(derived_words))
-        },
+        }
```

Comment:
```suggestion
        },
```
We chose to keep the coma on every match cases